guide/how-it-works/custom-spending-conditions.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: guide
+title: Custom Spending Conditions
+description: An overview about advanced capabilities that enable users to create more flexible bitcoin wallets.
+nav_order: 12
+parent: How it works
+permalink: /guide/how-it-works/custom-spending-conditions/
+main_classes: -no-top-padding
+image: /assets/images/guide/how-it-works/custom-spending-conditions/header.jpg
+image_base: /assets/images/guide/how-it-works/custom-spending-conditions/
+
+---
+
+<!--
+
+Editor's notes
+
+This page focuses on the advanced wallet configurations which are enabled by Miniscript.
+
+Header image source:
+https://www.figma.com/file/qzvCvqhSRx3Jq8aywaSjlr/Bitcoin-Design-Guide-Illustrations-CO?type=design&node-id=3324%3A24043&mode=design&t=GbcNUdc0VUnSOQi2-1
+
+-->
+
+{% include picture.html
+   image = "assets/images/guide/how-it-works/custom-spending-conditions/header.jpg"
+   retina = "assets/images/guide/how-it-works/custom-spending-conditions/header@2x.jpg"
+   alt-text = ""
+   width = 1600
+   height = 800
+   layout = "full-width"
+%}
+
+
+# Custom Spending Conditions
+{:.no_toc}
+---
+
+Spending conditions encode the rules that need to be followed by a wallet to spend funds. Most bitcoin wallets have only one spending condition. For example, in a single-key wallet there is only one key that can sign transactions. Or, in a 2-of-3 multi-key wallet, you need any two of the three keys to sign transactions.
+
+In addition to these standard configurations it is also possible to create more flexible wallets by defining custom spending conditions. There is a wide variety of use cases, where this flexibility can be helpful, like personal or family self-custody, inheritance or company treasuries.
+
+## Use case examples
+
+#### Self-custody
+
+Custom spending conditions can be helpful to decrease the risk of losing funds in personal self-custody context. In a savings wallet context where you are the only key holder, you could unlock a recovery spending condition after a certain time of wallet inactivity. This condition could contain an additional key held in a different location. Such a setup does not decrease the wallet security, because the second key is locked under normal circumstances of using the wallet. 
```

Comment:
```suggestion
Custom spending conditions can be helpful to decrease the risk of losing funds in personal self-custody context. One example is in a savings wallet, wallet inactivity could unlock a recovery spending condition after a certain amount of time that a wallet is inactive. This condition could contain an additional key held in a different location. Such a setup does not decrease the wallet security, because the second key is locked under normal circumstances of using the wallet. 
```

Reply:
Agree that this sentence is sub-optimal currently. The suggestion still doesn't "sound" quite right. What do you think about something more tangible: 

"Custom spending conditions can be helpful to decrease the risk of losing funds in personal self-custody context.. **Let's say that you have a wallet that you use regularly on a weekly basis. You could unlock a recovery spending condition automatically if that wallet has not been used for more than six months, because it's likely that you have lost access to your primary key.** This condition could contain an additional key held in a different location.."

Reply:
I'd alm

```suggestion
Custom spending conditions can be helpful to decrease the risk of losing funds in personal self-custody context. Let's say that you have a wallet that you use regularly on a weekly basis. You could unlock a recovery spending condition automatically if that wallet has not been used for more than six months, because it's likely that you have lost access to your primary key. This condition could contain an additional key held in a different location. Such a setup does not decrease the wallet security, because the second key is locked under normal circumstances of using the wallet. 
```

Reply:
This reads so much better. Really like it.